SRC/seleniumdemotest.py

```diff
@@ -1,48 +1,68 @@
 from selenium import webdriver
 import time
 import pytest
+from selenium.common import NoSuchElementException
 from selenium.webdriver.common.by import By
 from selenium.webdriver.support import expected_conditions
 from selenium.webdriver.common.keys import Keys
 import random
+WEBSITE_LINK = "http://seleniumdemo.com/?page_id=7"
```

Comment:
Enter here would be nice. Try to find a quideline on how to format your code files and be consistent with it